static/app/views/starfish/types.tsx

```diff
@@ -27,6 +27,7 @@ export enum SpanMetricsField {
   SPAN_GROUP = 'span.group',
   SPAN_DURATION = 'span.duration',
   SPAN_SELF_TIME = 'span.self_time',
+  PROJECT = 'project',
```

Comment:
Could you add a note here that this is the ID, and not the slug? Same in `SpanIndexedField` below

Reply:
My bad! In that case, leave a note that it's the slug!